core/src/main/scala/kafka/server/DelegationTokenManager.scala

```diff
@@ -317,7 +317,7 @@ class DelegationTokenManager(val config: KafkaConfig,
               tokenInfo.setExpiryTimestamp(expiryTimeStamp)
 
               updateToken(token)
-              info(s"Delegation token renewed for token : " + tokenInfo.tokenId + " for owner :" + tokenInfo.owner)
+              info(s"Delegation token renewed for token : ${tokenInfo.tokenId} for owner :${tokenInfo.owner}")
```

Comment:
Existing issue, but we can maybe fix it now, there should be a space after the `:` not before. A few more cases in this file.